TADashboard/src/main/java/page_objects/DataProfilesGeneralSettingsPage.java

```diff
@@ -25,4 +27,8 @@ public void createNewProfile(String name, String type, String relatedData) {
         cbbRelatedData.select(relatedData);
         btnFinish.click();
     }
+
+    public boolean areItemTypesDisplayedInOptions(List<String> listItems) {
```

Comment:
you should get the options list then compare in the test script.

Reply:
Yeah, Okie. I got it

Reply:
I have updated code. Please check again.

![image](https://user-images.githubusercontent.com/56109274/134135727-f18eeb00-c930-4bae-a721-38f36e420a85.png)